### ISSUE TITLE
Sometimes the render doesn't work

### ISSUE THREAD

**Issue by mrpvb-pvbk** [asjqkkkk/markdown_widget#36]
**Describe the bug**
I don't know why, but sometimes it happens with the same data.

I am using this code:
```
SliverList(
                    delegate: SliverChildListDelegate(MarkdownGenerator(
                      data: snapshot.data,
                      childMargin:
                          EdgeInsets.symmetric(horizontal: 10.0, vertical: 5.0),
                    ).widgets),
                  ),
```
And this source data
[https://app.coder-side.ru/assets/books/coder_book/php/base/50.md](https://app.coder-side.ru/assets/books/coder_book/php/base/50.md)

**Screenshots**
![Screenshot_20200610-220622](https://user-images.githubusercontent.com/44783405/84355428-460a1c80-abcb-11ea-8214-ac6e602aabc5.jpg)

**Environment(please complete the following information):**
![Screenshot (11)](https://user-images.githubusercontent.com/44783405/84356442-dc8b0d80-abcc-11ea-8a8a-bfff110ac393.png)

**Platform**
Does the problem appear on the **Web** or on the **Mobile**？
This is mobile.
When I used the online demo everything was fine.

**Additional context**
This is rare, but it happens. And the source with which this happens is always the same. It doesn't happen that a file works and then it doesn't.

**Comment by asjqkkkk** [comment]
I can't read your md file, can you provider another one?

**Comment by mrpvb-pvbk** [comment]
Yes! 
[1](https://app.coder-side.ru/assets/books/coder_book/php/base/52.md)
[2](https://app.coder-side.ru/assets/books/coder_book/php/base/50.md)
[3](https://app.coder-side.ru/assets/books/coder_book/php/base/54.md)

**Comment by asjqkkkk** [comment]
![image](https://user-images.githubusercontent.com/30992818/84363953-07438900-ac02-11ea-805d-4b767fd40258.png)

maybe the way you use MarkdownGenerator is wrong, I can't find where the problem is😂

**Comment by mrpvb-pvbk** [comment]
I need use MarkdownGenerator for SliverList. I will create project for this.

**Comment by mrpvb-pvbk** [comment]
I cant find where is problem from.
I added you to my [repository](https://github.com/XIIIFOX/Coder-Side/). Then download it. Run the app and give it permission on the memory.
Then follow these steps:
1. On the home screen, select the purple button(chip) with the name C#
2. To the left of the room. Find the number 6 and go.
3. Here is an example of how the Header was not rendered for some reason.

**Comment by asjqkkkk** [comment]
> I cant find where is problem from.
> I added you to my [repository](https://github.com/XIIIFOX/Coder-Side/). Then download it. Run the app and give it permission on the memory.
> Then follow these steps:
> 
> 1. On the home screen, select the purple button(chip) with the name C#
> 2. To the left of the room. Find the number 6 and go.
> 3. Here is an example of how the Header was not rendered for some reason.

Hi, don't show other people your private repository
You can describe your problem in detail, and then I take time to solve it

**Comment by mrpvb-pvbk** [comment]
> > I cant find where is problem from.
> > I added you to my [repository](https://github.com/XIIIFOX/Coder-Side/). Then download it. Run the app and give it permission on the memory.
> > Then follow these steps:
> > 
> > 1. On the home screen, select the purple button(chip) with the name C#
> > 2. To the left of the room. Find the number 6 and go.
> > 3. Here is an example of how the Header was not rendered for some reason.
> 
> Hi, don't show other people your private repository
> You can describe your problem in detail, and then I take time to solve it

Everything all right. I made a read permission. To describe the details, I need to understand where the problem is coming from. And I can't do that. Please check.

**Comment by asjqkkkk** [comment]
what dose it do?
```
visualDensity: VisualDensity.adaptivePlatformDensity,
```
my flutter version is dev-1.16.3
I can't use this property, so I removed it

**Comment by mrpvb-pvbk** [comment]
Where is it?

**Comment by mrpvb-pvbk** [comment]
So that you don't get confused the  markdown widget is located along this path: lib/main_viewer.dart

**Comment by asjqkkkk** [comment]
![image](https://user-images.githubusercontent.com/30992818/84381080-0bc86b80-ac1b-11ea-9abc-19a987423c37.png)

Is your markdown data not rendered correctly?

**Comment by mrpvb-pvbk** [comment]
> ![image](https://user-images.githubusercontent.com/30992818/84381080-0bc86b80-ac1b-11ea-9abc-19a987423c37.png)
> 
> Is your markdown data not rendered correctly?

Check it data in my app.

**Comment by mrpvb-pvbk** [comment]
> I cant find where is problem from.
> I added you to my [repository](https://github.com/XIIIFOX/Coder-Side/). Then download it. Run the app and give it permission on the memory.
> Then follow these steps:
> 1. On the home screen, select the purple button(chip) with the name C#
> 2. To the left of the room. Find the number 6 and go.
> 3. Here is an example of how the Header was not rendered for some reason.
>  

Use this.

**Comment by asjqkkkk** [comment]
> > I cant find where is problem from.
> > I added you to my [repository](https://github.com/XIIIFOX/Coder-Side/). Then download it. Run the app and give it permission on the memory.
> > Then follow these steps:
> > 
> > 1. On the home screen, select the purple button(chip) with the name C#
> > 2. To the left of the room. Find the number 6 and go.
> > 3. Here is an example of how the Header was not rendered for some reason.
> 
> Use this.

your project import too many other packages, I can't run in my enviroment.
If you can provider a minimal project, I can take less time to sovle this issue

**Comment by mrpvb-pvbk** [comment]
ok

**Comment by mrpvb-pvbk** [comment]
When I opened the page where there is a problem with rendering again I got this log:
```
I/flutter (32600): Uncatch Node:p
I/flutter (32600): Uncatch Node:b
I/flutter (32600): Uncatch Node:p
I/flutter (32600): Uncatch Node:i
I/flutter (32600): Uncatch Node:p
I/flutter (32600): Uncatch Node:b
I/flutter (32600): Uncatch Node:p
I/flutter (32600): Uncatch Node:i
```
What does it mean?

**Comment by asjqkkkk** [comment]
> When I opened the page where there is a problem with rendering again I got this log:
> 
> ```
> I/flutter (32600): Uncatch Node:p
> I/flutter (32600): Uncatch Node:b
> I/flutter (32600): Uncatch Node:p
> I/flutter (32600): Uncatch Node:i
> I/flutter (32600): Uncatch Node:p
> I/flutter (32600): Uncatch Node:b
> I/flutter (32600): Uncatch Node:p
> I/flutter (32600): Uncatch Node:i
> ```
> 
> What does it mean?

This is because text not rendering to widget correctly, it will print more details in v1.2.1

**Comment by mrpvb-pvbk** [comment]
> your project import too many other packages, I can't run in my enviroment.
> If you can provider a minimal project, I can take less time to sovle this issue

Finlay a way i made a simple project. It should help you find the error.
[https://github.com/mrpvb-pvbk/issue_md](https://github.com/mrpvb-pvbk/issue_md)

**Comment by mrpvb-pvbk** [comment]
Any suggestions?😳

**Comment by asjqkkkk** [comment]
> Any suggestions?😳

sorry, I'm busy in my new job recently, I'll have a look for your project later

**Comment by asjqkkkk** [comment]
> Any suggestions?😳

It's strange, when I copy source data, and put it to editor, everything is ok:
![image](https://user-images.githubusercontent.com/30992818/86123744-97dbfd80-bb0c-11ea-82e3-ae3dbfae7813.png)


maybe the data you download had been changed

I need time to check it

**Comment by asjqkkkk** [comment]
I found the problem, I'll solve this issue tonight😄

**Comment by asjqkkkk** [comment]
Hi, you can try for 
```
  markdown_widget:
    git:
      url: https://github.com/asjqkkkk/markdown_widget.git
      ref: dev
```

if this issue is solved, I'll update to v1.2.2

**Comment by mrpvb-pvbk** [comment]
Finally working)
